projects/packages/search/src/dashboard/components/form-toggle/compact.jsx

```diff
@@ -9,10 +8,7 @@ export default class CompactFormToggle extends Component {
 
 	render() {
 		return (
-			<Toggle
-				{ ...omit( this.props, 'className' ) }
-				className={ clsx( this.props.className, 'is-compact' ) }
-			>
+			<Toggle { ...this.props } className={ clsx( this.props.className, 'is-compact' ) }>
```

Comment:
Heh, I wonder what the thought here was, since literally they were removing the className and adding it back derived from the same source...

Reply:
Maybe someone didn't realize it worked? Or maybe an old version didn't like the duplicates? 🤷